tests/test_multidict.py

```diff
@@ -281,34 +281,72 @@ def test_keys_is_set_less(self, cls: Type[MutableMultiMapping[str]]) -> None:
 
         assert d.keys() < {"key", "key2"}
 
-    def test_keys_is_set_less_equal(self, cls: Type[MutableMultiMapping[str]]) -> None:
-        d = cls([("key", "value1")])
+    @pytest.mark.parametrize(
+        ("contents", "expected"),
+        (
+            ([("key", "value1")], True),
+            ([("key", "value1"), ("key2", "value2")], True),
+            ([("key", "value1"), ("key2", "value2"), ("key3", "value3")], False),
+            ([("key", "value1"), ("key3", "value3")], False),
+        ),
+    )
+    def test_keys_is_set_less_equal(
+        self,
+        cls: Type[MutableMultiMapping[str]],
+        contents: List[Tuple[str, str]],
+        expected: bool,
+    ) -> None:
+        d = cls(contents)
 
-        assert d.keys() <= {"key"}
+        assert (d.keys() <= {"key", "key2"}) == expected
```

Comment:
For booleans, it is customary to use an identity check, just like with `None`:
```suggestion
        assert (d.keys() <= {"key", "key2"}) is expected
```

Although, I'd probably put the check result into a variable to avoid having complex nested inline code like this.